client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-footer.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { actionType } from './constants';
+
+class DomainConnectAuthorizeFooter extends Component {
+	static propTypes = {
+		isPlaceholder: PropTypes.bool,
+		onClose: PropTypes.func,
+		onConfirm: PropTypes.func,
+		showAction: PropTypes.oneOf( actionType )
+	};
+
+	static defaultProps = {
+		isPlaceholder: false
+	};
+
+	placeholder = () => {
+		return (
+			<div className="domain-connect__is-placeholder">
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	renderActionConfirmCancel = () => {
+		const { translate } = this.props;
+		const notReadyToSubmit = actionType.READY_TO_SUBMIT !== this.props.showAction;
+
+		return (
+			<div>
+				<p>
+					{
+						translate( 'When you\'re ready to proceed, click Confirm. If this isn\'t what you meant to do, ' +
```

Comment:
A good thing might be to re-use the `Confirm` and `Cancel` labels - you have them in the same scope, so you can just pass as placeholders to this string. That way the translators won't have to translate them twice and there's no chance that they'll use a different translation for them.